tests/onnxruntime/test_modeling.py

```diff
@@ -1698,18 +1726,20 @@ def test_compare_to_transformers(self, model_arch):
         for k, v in inputs.items():
             inputs[k] = [v[i : i + num_choices] for i in range(0, len(v), num_choices)]
 
-        inputs = dict(inputs.convert_to_tensors(tensor_type="pt"))
-        onnx_outputs = onnx_model(**inputs)
+        pt_inputs = dict(inputs.convert_to_tensors(tensor_type="pt"))
+        with torch.no_grad():
+            transformers_outputs = transformers_model(**pt_inputs)
 
-        self.assertTrue("logits" in onnx_outputs)
-        self.assertIsInstance(onnx_outputs.logits, torch.Tensor)
+        for input_type in ["pt", "np"]:
+            inps = dict(inputs.convert_to_tensors(tensor_type=input_type))
+            onnx_outputs = onnx_model(**inps)
 
-        with torch.no_grad():
-            transformers_outputs = transformers_model(**inputs)
+            self.assertTrue("logits" in onnx_outputs)
+            self.assertIsInstance(onnx_outputs.logits, self.TENSOR_ALIAS_TO_TYPE[input_type])
 
-        # Compare tensor outputs
-        print(torch.max(torch.abs(onnx_outputs.logits - transformers_outputs.logits)))
-        self.assertTrue(torch.allclose(onnx_outputs.logits, transformers_outputs.logits, atol=1e-4))
+            # Compare tensor outputs
+            print(torch.max(torch.abs(torch.Tensor(onnx_outputs.logits) - transformers_outputs.logits)))
```

Comment:
I know it was like this before this PR, but why do we need to print this? It could probably be removed.